pages/glossary/instrument.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Instrument to achieve an objective
+key: instrument-to-achieve-an-objective
+unambiguous: true
+objective: false
+---
+
+An [HTML element][] that when [activated][] allows an end-user to achieve an objective. 
```

Comment:
You'll need to add an assumption to the rule that there is for this instrument, there is no other mechanism available to achieve the objectives.

Reply:
Done!